README.md

```diff
@@ -82,15 +82,17 @@ helium-gateway pre-installed, manual installation requires you to:
    Supported values are `stdio` or `syslog`. Note you may need to configure
    the `syslog` service on your device to accept the logs.
 
-6. Configure the region if required. The default region of the gateway is
-   `US915`, if your region is different you can set the right one in the
-   `settings.toml` file.
+6. Configure the region if required. The default region of the gateway is set to
+   `UKNOWN`, and fetched from the asserted location of the gateway. Setting the
+   regioon to a known region or caching the last fetched region and using the
```

Comment:
```suggestion
6. Configure the region. The default region of the gateway is set to
   `UKNOWN`, and fetched from the asserted location of the gateway. Setting the
   region to a known region or caching the last fetched region and using the
```
seems required now

Reply:
No.. the region is not required at all.. That's the whole point of not having it.. that there _is_ no good default region so jus too get the current asserted region from chain.